website/blog/2018-11-01-oss-roadmap.md

```diff
@@ -0,0 +1,67 @@
+
+## 🎁 Open Sourcing Internals and 🛠Updated Tooling
+
+The React Native development experience for product teams at Facebook can be quite different from open source. Tools that may be popular in the open source community are not used at Facebook. There may be an internal tool that achieves the same purpose. In some cases, Facebook teams have become used to tools that do not exist outside of Facebook. These disparities can pose challenges when we open source our upcoming architecture work.
+
+We'll work on releasing some of these internal tools. We'll also improve support for tools popular with the open source community. Here's a non-exhaustive list of projects we'll tackle:
+
+* Open source JSI and enable the community to bring their own JavaScript VMs, replacing the existing JavaScriptCore from RN's initial release.
+* Support 64-bit libraries on Android.
+* Enable debugging under the new architecture.
+* Improve support for CocoaPods, Gradle, Maven, and new Xcode build system.
+
+## ✅ Testing Infrastructure
+
+When Facebook engineers publish code, it's considered safe to land if it passes all tests. These tests identify whether a change might break one of our own React Native surfaces. Yet, there are differences in how Facebook uses React Native. This has allowed us to unknowingly break React Native in open source.
+
+We'll shore up our internal tests to ensure they run in an environment that is as close as possible to open source. This will help prevent code that breaks these tests from making it to open source. We will also work on infrastructure to enable better testing of the core repo on GitHub, enabling future pull requests to easily include tests.
+
+Combined with the reduced surface area, this will allow contributors to merge pull requests quicker, with confidence.
+
+## 📜 Public API
+
+Facebook will consume React Native via the public API, the same way open source does, to reduce unintentional breaking changes. We have started converting internal call sites to address this. Our goal is to converge on a stable, public API, leading to the adoption of semantic versioning in version 1.0.
+
+## 📣 Communication
+
+React Native is one of the [top open source projects on GitHub](https://octoverse.github.com/projects) by contributor count. That makes us really happy, and we'd like to keep it going. We'll continue working on initiatives that lead to involved contributors, such as increased transparency and open discussion. The documentation is one of the first things someone new to React Native will encounter, yet it has not been a priority. We'd like to fix that, starting with bringing back auto-generated API reference docs, and creating additional content focused on creating [quality user experiences](http://facebook.github.io/react-native/docs/improvingux).
```

Comment:
maybe also mention that we want to improve the changelog?

and maybe even link -> https://github.com/react-native-community/react-native-releases/issues/47